src/components/pages/Profile/EditProfile.js

```diff
@@ -1,33 +1,40 @@
 import React, { useState } from 'react';
-import { Form, Input, Button, Radio, Modal, TreeSelect } from 'antd';
+import { Form, Input, Button, Modal, TreeSelect } from 'antd';
 import '../../../styles/styles.css';
 import { connect } from 'react-redux';
-import useForms from '../../../hooks/useForms';
-
-const initialValues = {
-  first_name: 'Hal',
-  last_name: 'Jordan',
-  email: 'greenguy123@gmail.com',
-  location: 'Earth',
-  company: 'Bloom Tech, SWE',
-  tech_stack: 'React',
-  commitment: 'Pair Programming',
-};
-// Not showing actual initial values in table ( logged in users info )
+import axiosWithAuth from '../../../utils/axiosWithAuth';
 
 function EditProfile({ userInfo }) {
-  const [formValues, handleChange, clearForm] = useForms(initialValues);
+  const [form] = Form.useForm();
 
   const [ModalOpen, setModalOpen] = useState(false);
 
   const showModal = () => setModalOpen(true);
 
   const handleCancel = () => setModalOpen(false);
 
-  const handleSubmit = e => {
+  const onCreate = values => {
     setModalOpen(false);
-    console.log(formValues);
-    clearForm(e);
+    axiosWithAuth()
+      .put('/profile', values)
+      .then(res => {
+        form.setFieldsValue(values);
+      })
+      .catch(err => {
+        console.log(err);
```

Comment:
Please remove any `console.log`'s from the code base.

Reply:
I added some functionality to the error handling. I have posted another video in the PR that walks through the couple changes I made to the modal.